auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -273,6 +313,16 @@ export default class EntityManagement extends Mixins(AccountChangeMixin, NextPag
     return LaunchDarklyService.getFlag(LDFlags.EnableMandatoryAddress) || false
   }
 
+  private get enableBusinessTable (): boolean {
+    // disabling table manually for early stages of development.
+    return false // LaunchDarklyService.getFlag(LDFlags.EnableBusinessTable) || false
+  }
+
+  /** Is True if the current account is premium. */
+  private get isPremiumAccount (): boolean {
+    return this.currentOrganization?.orgType === Account.PREMIUM
```

Comment:
Is there a global getter for this? I suspect this is a frequent/common check in this UI.

Reply:
You are right, i think it's used in a few places now. Will make this global, in next commit 👍

Reply:
Updated in the latest commit